src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/AwakePage.xaml

```diff
@@ -29,86 +29,50 @@
-                                                        HorizontalAlignment="Left"
-                                                        MinWidth="90"
-                                                        SmallChange="1" 
-                                                        LargeChange="5"/>
-                                        <muxc:NumberBox x:Uid="Awake_TemporaryKeepAwake_Minutes"
-                                                        Value="{x:Bind ViewModel.Minutes, Mode=TwoWay}"
-                                                        IsEnabled="{x:Bind ViewModel.IsTimeConfigurationEnabled, Mode=OneWay}"
-                                                        Minimum="0"
-                                                        SpinButtonPlacementMode="Compact"
-                                                        Margin="8,0,0,0"
-                                                        HorizontalAlignment="Left"
-                                                        MinWidth="90"
-                                                        SmallChange="1" 
-                                                        LargeChange="5"/>
-                                    </StackPanel>
-                                </StackPanel>
+                                <muxc:NumberBox x:Uid="Awake_TemporaryKeepAwake_Hours"
+                                                Value="{x:Bind ViewModel.Hours, Mode=TwoWay}"
+                                                Minimum="0"
+                                                SpinButtonPlacementMode="Compact"
+                                                HorizontalAlignment="Left"
+                                                Width="96"
+                                                SmallChange="1" 
+                                                LargeChange="5"/>
+                                <muxc:NumberBox x:Uid="Awake_TemporaryKeepAwake_Minutes"
+                                                Value="{x:Bind ViewModel.Minutes, Mode=TwoWay}"
+                                                Minimum="0"
+                                                SpinButtonPlacementMode="Compact"
+                                                Margin="8,0,0,0"
+                                                HorizontalAlignment="Left"
+                                                Width="96"
+                                                Maximum="60"
+                                                SmallChange="1" 
+                                                LargeChange="5"/>
                             </StackPanel>
-                        </controls:SettingExpander.Content>
-                    </controls:SettingExpander>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+                    <controls:Setting x:Uid="Awake_EnableDisplayKeepAwake" Icon="&#xE7FB;">
```

Comment:
This icon doesn't really include the "awake" part of this setting... maybe something like `f08c` "BlueLight" ?